R/produce_sc_ch_episodes_tests.R

```diff
@@ -2,20 +2,22 @@
 #'
 #' @param data new or old data for testing summary flags (data is from \code{\link{get_sc_ch_episodes_path}})
 #'
-#' @return a dataframe with a count of each flag \code{\link{sum_test_flags}} and \code{\link{create_demog_test_flags}}
+#' @return a dataframe with a count of each flag \code{\link{sum_test_flags}} and
+#' \code{\link{create_demog_test_flags}}. Missing value flag from \code{\link{is_missing}}
+#'
 #' @export
 #' @importFrom dplyr mutate select
 #' @family produce tests functions
 produce_sc_ch_episodes_tests <- function(data) {
   data %>%
     # create test flags
     create_demog_test_flags %>%
-    dplyr::mutate(
-      n_missing_sending_loc = if_else(is.na(.data$sending_location) | .data$sending_location == "", 1, 0),
-      n_missing_sc_id = if_else(is.na(.data$social_care_id) | .data$social_care_id == "", 1, 0)
+    mutate(
+      n_missing_sending_loc = if_else(is_missing(.data$sending_location), 1, 0),
+      n_missing_sc_id = if_else(is_missing(.data$social_care_id), 1, 0)
     ) %>%
     # remove variables that won't be summed
-    dplyr::select(-c(.data$chi:.data$sc_latest_submission)) %>%
+    select(-c(.data$chi:.data$sc_latest_submission)) %>%
```

Comment:
Can you spell out variables please, either to keep or to drop? Using the `:` means that the order of the variables will always have to be the same.